locale/pt_BR/manager.po

```diff
@@ -3050,39 +3050,41 @@ msgstr "Tem certeza de que deseja continuar e excluir esta instituição?"
 msgid "manager.setup.statistics"
 msgstr "Estatísticas"
 
+#, fuzzy
 msgid "manager.settings.statistics.compressStatsLogs.description"
 msgstr "Escolha o que fazer com os arquivos de log arquivados."
 
+#, fuzzy
 msgid "manager.settings.statistics.compressStatsLogs.compress"
 msgstr "Compactar arquivos"
 
+#, fuzzy
 msgid "manager.settings.statistics.keepDaily"
 msgstr "Métricas diárias"
 
-msgid "manager.settings.statistics.keepDaily.option"
-msgstr "Manter as métricas diárias na base de dados"
-
+#, fuzzy
 msgid "manager.settings.statistics.geoUsageStats"
 msgstr "Estatísticas de uso geográfico"
 
-msgid "manager.settings.statistics.geoUsageStats.description"
-msgstr ""
-"Escolha quais dados geográficos de estatísticas de uso você gostaria de "
-"coletar. Isso influenciará os possíveis relatórios de estatísticas que você "
-"pode recuperar e também terá um impacto no tamanho da base de dados."
-
```

Comment:
@NateWr, it seems like those deletions are accidentally, or?

Reply:
I think I removed them because I thought the substance changed enough that we should force the re-translation (particularly the addition of a note on anonymity). But if you want to use the `#, fuzzy` for them instead that's ok.

Reply:
No, that is fine, just that the 'fuzzy' is added too...
So maybe I will remove the 'fuzzy'...

Reply:
Oh I see, two `#, fuzzy`s. I do that all the time by accident. :hand_over_mouth: